lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -117,6 +120,82 @@ def setup_students_and_grades(context):
                     )
 
 
+class TestAdminAccessCoachDashboard(CcxTestCase, LoginEnrollmentTestCase):
+    """
+    Tests for Custom Courses views.
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAdminAccessCoachDashboard, cls).setUpClass()
```

Comment:
There's no need for a method implementation that only calls super().  This seems to be done a lot in the CCX tests.  Is there a reason for it I don't understand?